stdlib/http/requests/requests.flux

```diff
@@ -183,6 +183,19 @@ do = (
 // > array.from(rows: [data])
 // ```
 //
+// ### Make a POST request with query parameters
+//
+// ```
+// import "http/requests"
+//
+// response = requests.post(
```

Comment:
I wonder if it would be overkill to add a comment inside the example to show the full url being generated: `http://example.com?start=100`

Reply:
Add the full URL in a comment. I also added a second param with multiple values to show how those get parsed into the URL.